src/test/java/com/browserstack/test/utils/BrowserstackTestStatusListener.java

```diff
@@ -30,7 +30,7 @@ public void onTestFailure(ITestResult result) {
         WebDriver driver = ((TestBase) currentClass).getDriver();
         String message = result.getThrowable().getMessage();
         String reason = (message != null && message.length() > 254) ? message.substring(0, 254) : message;
-        markTestStatus("failed", reason, driver);
+        markTestStatus("failed", reason.replaceAll("[^a-zA-Z0-9]", " "), driver);
```

Comment:
just thought about . (full stop) should we add that? It will make the sentences readable.